manim/animation/indication.py

```diff
@@ -581,19 +576,17 @@ class Circumscribe(Succession):
 
     Examples
     --------
-
     .. manim:: UsingCircumscribe
 
         class UsingCircumscribe(Scene):
             def construct(self):
-                lbl = Tex(r"Circum-\\\\scribe").scale(2)
+                lbl = Tex(r"Circum-\\scribe").scale(2)
```

Comment:
```suggestion
                lbl = Tex(r"Circum-scribe").scale(2)
```
That line break using `\\`causes rtd build failure